.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+<!--- Provide a general summary of the issue in the Title above -->
```

Comment:
Might be nice to add a comment here about having checked if a similar issue exists.

Reply:
Another thing might be to mention https://github.com/chentsulin/electron-react-boilerplate/issues/400

Reply:
:+1: